frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -462,6 +522,40 @@ function ReaderHighlight:addToMainMenu(menu_items)
             })
         end,
     })
+    if Screen:isColorScreen() then
+        table.insert(menu_items.highlight_options.sub_item_table, {
+            text = _("Apply default style to all highlights"),
+            enabled_func = function()
+                return Screen:isColorEnabled()
+            end,
+            callback = function(touchmenu_instance)
+                UIManager:show(ConfirmBox:new{
+                    text = _("Are you sure you want to edit all highlights."),
```

Comment:
("edit" feels to my non-native english ears that we'll go thru a process of editing one highlight after another. "update" sounds more like an automatic process)

Reply:
Update sounds good to me. I'll change it.